### ISSUE TITLE
Using multiple font faces 

### ISSUE THREAD

**Issue by jacksenior** [milesj/aesthetic#32]
When trying to set up multiple '@font-face' it seems to always load the font that is declared last and ignore the first.

Take this example
```
'@font-face': {
  Roboto: {
    fontStyle: 'normal',
    fontWeight: 'normal',
    srcPaths: ['fonts/Roboto.woff2', 'fonts/Roboto.ttf'],
  },
  Roboto2: {
    fontStyle: 'normal',
    fontWeight: 'normal',
    srcPaths: ['fonts/Roboto2.woff2', 'fonts/Roboto2.ttf'],
  },
}
```

In the network Tab I would get Roboto2 downloading and being used around the site but not Roboto.
Tried Lots of different implementations after seeing this [jss issue](https://github.com/cssinjs/jss/issues/199)

Unsure if its implementation or a limitation at this point?

**Comment by milesj** [comment]
So you're using JSS yeah?

Could you screenshot/paste me what the compiled CSS looks like by chance? And what the `font-family` on the element looks like?

**Comment by jacksenior** [comment]
The code for implementation is.
<img width="248" alt="screen shot 2018-05-17 at 19 43 44" src="https://user-images.githubusercontent.com/2884464/40197214-ad8999f4-5a0a-11e8-9b6d-6f5d4ac47135.png">

<img width="513" alt="screen shot 2018-05-17 at 19 43 05" src="https://user-images.githubusercontent.com/2884464/40197221-b2857298-5a0a-11e8-96c8-58ec7d2de017.png">

The compiled css in the DOM is
<img width="571" alt="screen shot 2018-05-17 at 19 42 45" src="https://user-images.githubusercontent.com/2884464/40197243-c4c387ec-5a0a-11e8-9093-14422f5e9005.png">

Stripped the Font stack on Html back to just the imported for testing.
<img width="322" alt="screen shot 2018-05-17 at 19 45 21" src="https://user-images.githubusercontent.com/2884464/40197287-f04f2010-5a0a-11e8-9d88-9a221da55385.png">

**Comment by milesj** [comment]
Awesome thanks, will look into.

That `data-meta` also looks suspect.

**Comment by milesj** [comment]
Bug has been fixed in 2.3.2, thanks for the report!
